.github/workflows/build-book-release.yml

```diff
@@ -134,6 +134,7 @@ jobs:
           ## Ideally, all dependencies should get installed in the first pass.
           # Remove when 0.60.1 available
           ## BiocManager::install("Cairo")
+          remotes::install_github("LTLA/bluster")
```

Comment:
I pushed updates to Bioc devel just couple days ago, and they are not there yet. After changes are there (in couple of days), this line should be useless

Reply:
If this is not urgent, we could wait so that there is not too many additional lines

bluster should be then imported with mia package